File: lv2/12951.py
'''
JadenCase 문자열 만들기
https://school.programmers.co.kr/learn/courses/30/lessons/12951
'''
def solution(s):
  arr = s.lower().split(' ')
  return ' '.join(['' if a == '' else a[0].upper() + a[1:] for a in arr])

File: lv2_js/12951.js
`
JadenCase 문자열 만들기
https://school.programmers.co.kr/learn/courses/30/lessons/12951
`
function solution(s) {
  const arr = s.toLowerCase().split(' ')  //모든 문자를 소문자로 변경한 뒤 공백을 기준으로 나눠주고
    .map(v => v.length > 1 ? v[0].toUpperCase() + v.slice(1,) : v.toUpperCase()); // 나눈 값이 1글자라면 대문자로 변경하고 2글자부터는 첫번째만 대문자로 변경하고 나머지 글자는 이어 붙인다
  return arr.join(' ');
}